### ISSUE TITLE
Error: unexpected Token: /*

### ISSUE THREAD

**Issue by Frozen-byte** [akserg/ng2-toasty#84]
* **I'm submitting a ...**
bug report

* **What is the current behavior?**
After running "npm run build" the compiler does strange things to your compiled toasty.utils.js.
The first comment is duplicated for unknown reasons:
toasty.utils.ts:
```
/**
 * Check and return true if an object is type of string
 */
export function isString(obj:any) {
    return typeof obj === "string";
}
```
toasty.utils.js:
```
/**
 * Check and return true if an object is type of string
 */
/**
 * Check and return true if an object is type of string
 */ export function isString(obj) {
    return typeof obj === "string";
}
```

* **If the current behavior is a bug, please provide the steps [...].**
run `npm run build` and open `toasty.utils.js`

* **What is the expected behavior?**
the corruption prevents rollup@0.42.0 to bundle the files:
```
[12:09:28] Error: Command failed: node --max_old_space_size=4096 ./node_modules/.bin/rollup -c rollup.config.js
Error: Unexpected token
node_modules/ng2-toasty/src/toasty.utils.js (1:1)
1:  */ export function isString(obj) {
    ^
2:     return typeof obj === "string";
3: }
```
* **Please tell us about your environment:**

- Angular version: "^4.1.3",
- Browser: none

**Comment by Frozen-byte** [comment]
To fix this, I currently add a simple blank line at line 1 to `toasty.utils.ts`...

**Comment by akserg** [comment]
Hi @Frozen-byte 

Never seen that issue before. Thanks for the advice. I added a simple blank line at line 1 to `toasty.utils.ts` as well. Fix available in 4.0.3.
